x-pack/plugins/cases/public/components/all_cases/table_filters.tsx

```diff
@@ -47,6 +47,7 @@ const defaultInitial = {
   reporters: [],
   status: StatusAll,
   tags: [],
+  owner: [],
```

Comment:
Kind of the same thing here. I figure when you're in a solution, everything listed here, but the owner will actually be modifiable by the user. You can just skip passing it through here and call it directly in the `useGetTags` and `useGetReporters` calls